sandbox/modules/identity/src/test/java/org/opensearch/identity/remotecluster/UserIT.java

```diff
@@ -112,4 +114,106 @@ public void testInternalUsersApi() throws Exception {
 
     }
 
+    public void testResetPasswordApi() throws Exception {
```

Comment:
This test is doing a lot, can this be broken down into smaller tests?

Reply:
I have refactored them into three cases.